src/pages/Home/Home.scss

```diff
@@ -0,0 +1,20 @@
+.home {
+  width: 100%;
+  padding: 1rem;
+  text-align: center;
+  max-width: 64rem;
+  margin: 0 auto;
+
+  h1 {
```

Comment:
You shouldn't give style to native elements, instead give classes to the elements